test/unit/testApplicantAddress.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const initSteps = require('app/core/initSteps');
+const {expect} = require('chai');
+const steps = initSteps([`${__dirname}/../../app/steps/action/`, `${__dirname}/../../app/steps/ui`]);
+const ApplicantAddress = steps.ApplicantAddress;
+
+describe('ApplicantAddress', () => {
+    describe('getUrl()', () => {
+        it('should return the correct url', (done) => {
+            const url = ApplicantAddress.constructor.getUrl();
+            expect(url).to.equal('/applicant-address');
+            done();
+        });
+    });
+
+    describe('getContextData()', () => {
+        let ctx;
+        it('should return the ctx with the deceased addresses', (done) => {
```

Comment:
This test is testing the applicant not the deceased so can you update the test description

Reply:
done!